umap/static/umap/test/URLs.mjs

```diff
@@ -0,0 +1,48 @@
+import URLs from "../js/modules/urls.mjs";
+
+describe('URLs', () => {
```

Comment:
Quotes are sometimes single or double and I'm not sure what rule applies in your code